releases/release-1.31/release-notes/maps/pr-122025-map.yaml

```
@@ -0,0 +1,4 @@
+pr: 122025
+releasenote:
+  text: `lastSuccessfullTime` in cronjobs will now be set reliably.
```

Having an initial backtick invalid yaml and will break: https://github.com/kubernetes/release/issues/2753

```suggestion
  text: "`lastSuccessfullTime` in cronjobs will now be set reliably."
```